src/Sylius/Bundle/ChannelBundle/Resources/config/driver/doctrine/orm.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Paweł Jędrzejewski
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services
+                               http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <parameters>
+        <parameter key="sylius.repository.channel.class">Sylius\Bundle\ChannelBundle\Doctrine\ORM\ChannelRepository</parameter>
```

Comment:
If ORM driver is set, then we will use this repository. Otherwise EntityRepository will be used.